### ISSUE TITLE
Implement krb5_set_trace_filename()

### ISSUE THREAD

**Issue by spaceone** [heimdal/heimdal#281]
It would be nice if "krb5_set_trace_filename" would be implemented which makes debugging pam_krb5 (libpam-heimdal) easier. See also: https://k5wiki.kerberos.org/wiki/Projects/Trace_logging

**Comment by nicowilliams** [comment]
Yes, we're quite interested in building such a feature at some point.  We want tracing/logging to be machine-parseable too, so that we might build diagnostic tools that consume such data.
